components/script/dom/gpucanvascontext.rs

```diff
@@ -199,6 +203,12 @@ impl GPUCanvasContext {
             HTMLCanvasElementOrOffscreenCanvas::OffscreenCanvas(canvas) => canvas.get_size(),
         }
     }
+
+    pub fn recreate(&self) {
+        if let Some(config) = self.config.take() {
+            self.Configure(&config)
+        }
```

Comment:
Why isn't the call `Unconfigure()` here instead of in `Configure()`?

Reply:
That's how it was done in mozilla, but I will recheck specs tomorrow.